src/pythie_serving/table_wrapper.py

```diff
@@ -0,0 +1,82 @@
+import os
+import logging
+
+import grpc
+import numpy as np
+import json
+
+from .tensorflow_proto.tensorflow_serving.config import model_server_config_pb2
+from .tensorflow_proto.tensorflow_serving.apis import predict_pb2, prediction_service_pb2_grpc
+from .utils import make_ndarray_from_tensor
+from .exceptions import PythieServingException
+import csv
+
+
+class TablePredictionServiceServicer(prediction_service_pb2_grpc.PredictionServiceServicer):
+
+    def __init__(self, *, logger: logging.Logger, model_server_config: model_server_config_pb2.ModelServerConfig):
+        self.logger = logger
+        self.table_map = {}
+        for table_config in model_server_config.model_config_list.config:
+            with open(os.path.join(table_config.base_path, "metadata.json"), "r") as f:
```

Comment:
use the new python interface pathlib https://docs.python.org/fr/3/library/pathlib.html and check existence with Path.existence